src/main/kotlin/com/terraformation/backend/seedbank/api/AccessionsController.kt

```diff
@@ -117,10 +118,11 @@ data class AccessionHistoryEntryPayload(
     @Schema(description = "User-entered notes about the event, if any.") //
     val notes: String?,
     val type: AccessionHistoryType,
+    val batchId: BatchId?,
```

Comment:
Move this up so the field list stays alphabetical. It could also use a description.